sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/configuration/AadOAuth2ClientConfiguration.java

```diff
@@ -95,11 +110,13 @@ OAuth2ClientAuthenticationJwkResolver oAuth2ClientAuthenticationJwkResolver(AadA
      */
     @Bean
     @ConditionalOnMissingBean
-    OAuth2AuthorizedClientManager authorizedClientManager(ClientRegistrationRepository clientRegistrations,
-                                                                 OAuth2AuthorizedClientRepository authorizedClients,
-                                                                 RefreshTokenOAuth2AuthorizedClientProvider refreshTokenProvider,
-                                                                 JwtBearerOAuth2AuthorizedClientProvider jwtBearerProvider,
-                                                                 ObjectProvider<OAuth2ClientAuthenticationJwkResolver> jwkResolvers) {
+    OAuth2AuthorizedClientManager authorizedClientManager(
```

Comment:
Shall we override the same bean definition for the Azure AD B2C side since the proxy ability?

Reply:
Will do. 

BTW, Azure AD B2C starter does not support using cert instead of client-secret now?

Reply:
Yes.